cmd/annotations_prep/main.go

```diff
@@ -440,38 +451,55 @@ func generateVariableName(annoName string) string {
 }
 
 func getFeatureStatus(fs string) (FeatureStatus, error) {
-	switch FeatureStatus(strings.ToTitle(fs)) {
+	asTitle := strings.Title(fs)
+	switch FeatureStatus(asTitle) {
```

Comment:
Maybe add Experimental as another one status?

Reply:
I'd like to keep that separate for now ... @brian-avery is working on expanding feature status.